### DISCUSSION TITLE
type error: property 'session' does not exist on type 'supabaseauthclient'. ts2339

### DISCUSSION THREAD

**Discussion by Carlintheghoul120** [supabase/supabase#9633] [Questions]
Hi I'm trying to check for a bug in one ionic react application by creating another one I'm starting with authentication and redirecting as the bug involves a blank page which happens to be the tab that is opened after redirecting (works fine in browser but is blank in android emulator) while the others work fine(even in the emulator). Anyway the user, session and signIn functions do not show up eg. Supabase.auth.user(); and show similar errors as the one mentioned in the title, I've tried uninstalling the dependency and reinstalling. Removed node modules and reinstalled but problem persists

**Comment by GaryAustin1** [comment]
You probably are running Supabase-js v2 with v1 code.   Those 3 functions do not exist in v2. 
https://supabase.com/docs/reference/javascript/release-notes

**Reply by Carlintheghoul120** [reply]
will check this out thank you
